.clippy.toml

```diff
@@ -5,4 +5,9 @@ disallowed-methods = [
     # see https://github.com/rust-lang/rust/issues/90308 for details.
     "std::env::set_var",
     "std::env::remove_var",
-]
\ No newline at end of file
+
+    # Avoid instances of https://github.com/rust-lang/rust/issues/86470
+    "std::time::Instant::duration_since",
+    "std::time::Instant::elapsed",
+    "std::time::Instant::sub",
```

Comment:
we can't actually detect uses of `sub` via clippy, afaict

Reply:
I believe that's why we've missed the `Uptime` timestamp formatter for the logs, which uses `-` (and clippy maybe can't detect banned methods that are called via operators?): https://github.com/linkerd/linkerd2-proxy/blob/d9dbdece1edf765b6c8d9ccdd16ca611c70c8669/linkerd/tracing/src/uptime.rs#L24